libc/benchmarks/CMakeLists.txt

```diff
@@ -202,4 +207,4 @@ target_link_libraries(libc.benchmarks.memory_functions.opt_host
 )
 llvm_update_compile_flags(libc.benchmarks.memory_functions.opt_host)
 
-add_subdirectory(automemcpy)
+add_subdirectory(automemcpy)
```

Comment:
Guessing the IDE ate the ending newline?

Reply:
Yeah, but I think I figured it out